test/ejabberd_loglevel_SUITE.erl

```diff
@@ -136,7 +135,6 @@ ejabberd_loglevel_running() ->
     application:set_env(lager, handlers, [Backend]),
     ejabberd_loglevel:init(),
     FileBackend = {BackendName, File},
-    true = timeout /= get_at_least_n_log_lines(File, length(Before) + 1, timer:seconds(5)),
```

Comment:
Why is it removed?

Reply:
It's all in the commit message: 9270cd463e768c41a3afb5031f8e3c08817a9c9e Basically lager on Erlang/21 doesn't write anything to the file just after starting the backend. Also I don't think this assertion is needed. We check content of the file later in tests anyway and there the file contains what we put to logs.